### PR TITLE
adding machine learning specific guide

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

**Comment by agitter** [comment]
This will be very helpful.  It is probably out of scope for this pull request, but eventually it will be nice to extend this to say something about high-throughput machine learning.  How can one use CHTC to sweep over different models, model configurations (hyperparameters), data splits, etc.?

**Comment by lmichael107** [comment]
Agreed. At the very least, we can add a sentence that also links to our guide for submitting many jobs at once, varying arguments and/or input files. @ChristinaLK ?

**Comment by ChristinaLK** [comment]
Get in your comments - I'll merge first thing tomorrow morning.

File: machine-learning-htc.shtml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+you can take advantage of CHTC's capability to run many independent jobs at once, 
+especially when each job is using a CPU. See our guide for running multiple jobs here: 
+
+* [Submitting Multiple Jobs Using HTCondor](multiple-jobs.shtml)
+
+# 2. Software Options
+
+Many of the tools used for machine learning, specifically deep learning and 
+convolutional neural networks, are sufficiently complex that our standard software 
+guides
```
Comment (agitter) [inline machine-learning-htc.shtml]: Some text is repeated here.  I read through quickly, and everything else here looks good enough to me for a first pass.

Comment (ChristinaLK) [reply machine-learning-htc.shtml]: oops, good catch - definitely started to edit a sentence there and didn't finish!

**Pull request merged**